python/ct/client/db/sqlite_cert_db.py

```diff
@@ -87,19 +77,19 @@ def __store_cert(self, cert, index, log_key, cursor):
                                     ('?,' * len(self.cert_table_columns))[:-1]),
```

Comment:
This looks nice and flexible, but I think that while it made sense with the original more dynamic approach (```[cert[column] for column in self.cert_table_columns]```), now that you do ```str(cert.version + 1), cert.serial_number``` directly (which I kind of like better, really), it doesn't make much sense. You've got an explicit mapping, and now, changing code far away from this method will break it, which is annoying.